src/styles/lib.less

```diff
@@ -16,3 +16,34 @@
     font-size: unit(@value * 16, px);
     font-size: @value;
 }
+
+.reset-list() {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.pointer() {
+    cursor: pointer;
+	cursor: hand;
```

Comment:
One is for IE the rest for everything else or, do we not care about that anymore ;)